### PR TITLE
MBS-8690: Show user fullname in tooltip

### LINKED ISSUE #19: Assigned users on iPad
As a student working with the iPad (Safari), I have no chance to find out the name of who is assigned to the card. I can see the user image, but the hover method does not work on the iPad.

### PR DESCRIPTION
**Pull request by sh-csg**
Fixes #19

### CODE REVIEW

File: templates/user.mustache [hunk 1]
```diff
@@ -26,6 +26,14 @@
...
\ No newline at end of file
+</div>
+{{#js}}
+    document.getElementById('mod_kanban_assignee-{{cardid}}-{{id}}').addEventListener('click', function(e) {
+        $('#mod_kanban_assignee-{{cardid}}-{{id}}').tooltip('show');
+        setTimeout(function() {
+            $('#mod_kanban_assignee-{{cardid}}-{{id}}').tooltip('hide');
+        }, 2000);
+    });
+{{/js}}
```
Comment (PhMemmel) [inline templates/user.mustache:39]: new line at end of file please.

Could you please put the JS code into a separate module? JS code here really should be as minimalistic as possible (which basically is "load module") due to coding guidelines AFAIK

Comment (sh-csg) [reply templates/user.mustache]: As the Moodle tooltip function relies on jQuery, I have to use it here too. Alternative would be to build an own tooltip function.

**Review by PhMemmel — COMMENTED** [review]
Looks good from both code and testing.

Just some minor thoughts:
It's a bit uncommon that the tooltip disappears after some time. The user probably would expect that he can hide the tooltip by
  - clicking again on the user picture
  - clicking somewhere outside the user picture
Are the technical obstacles why this has been implemented with a timeout?

If the timeout should be kept (which is totally fine), is it easily possible to add the other mentioned possibilities to hide the tooltip as well?

**Comment by sh-csg** [comment]
Changed way to trigger tooltip: Can be click (for touch devices), hover or focus now.

**Comment by PhMemmel** [comment]
Tested. Unfortunately tooltips do not show in Firefox DevTools.

It works on Chrome, is it possible to implement hiding the tooltip when click again on the user icon? Or would this be inconsistent with the defaults how the bootstrap tooltips usually work?

**Comment by sh-csg** [comment]
Added tabindex so it can gain focus also in Firefox. The behavior is the standard tooltip behavior in Bootstrap.

**Review by PhMemmel — APPROVED** [review]
LGTM, I like that it's now using bootstrap functionality with minimal changes on kanban's side :)

**Pull request merged**